src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -319,7 +334,7 @@ private void AddClaimsFromJToken(List<Claim> claims, string claimType, JToken jt
 
         private void AddDefaultClaimFromJToken(List<Claim> claims, string claimType, JToken jtoken, string issuer)
         {
-            JValue jvalue = jtoken as JValue;
+            var jvalue = jtoken as JValue;
             if (jvalue != null)
```

Comment:
These two lines can be consolidated into one. For example: if (jtoken is JValue jvalue)

Reply:
Will make this change. Thanks!